docs/serving/samples/autoscale-go/README.md

```diff
@@ -118,7 +118,7 @@ A demonstration of the autoscaling capabilities of a Knative Serving Revision.
 
 Knative Serving autoscaling is based on the average number of in-flight requests
 per pod (concurrency). The system has a default
-[target concurrency of 100](https://github.com/knative/serving/blob/master/config/core/configmaps/autoscaler.yaml#L54-L60)
+[target concurrency of 100](https://github.com/knative/serving/blob/8c7134ebfe368ee178229e99166e5bff8167b8ed/config/core/configmaps/autoscaler.yaml#L74)
```

Comment:
I'm nervous about pinning this to something which might be out of date (and therefore, future readers might miss that the default had changed again).

What do you think about either changing this to "master" or the `{{< branch >}}` shortcode?

Reply:
I'd be ok to even to link to just master branch filename and a string to search in it.

Reply:
@vagababov I'm not sure what that looks like, how do I post a URL with the correct search string?

Reply:
I meant, just to the file itself and a text to search, as in `look for X in this file for actual default value`.